.golangci.yml

```diff
@@ -34,4 +33,9 @@ linters:
     - unparam           # Reports unused function parameters
     - unused            # Checks Go code for unused constants, variables, functions and types
     - varcheck          # Finds unused global variables and constants [fast: true, auto-fix: false]
-    - whitespace        # Tool for detection of leading and trailing whitespace [fast: true, auto-fix: true]
\ No newline at end of file
+    - whitespace        # Tool for detection of leading and trailing whitespace [fast: true, auto-fix: true]
+
+linters-settings:
+  staticcheck:
+    go: "1.16"
+    checks: [ "all", "-SA1029"]
```

Comment:
What the purposes of this block?

Reply:
This block close problem which  linter send.